test/llvm/alignment/check-align-padding.chpl

```diff
@@ -38,8 +43,18 @@ record int64aligned16b {
   var field2: aligned16test;
 }
 
-proc compute(arg) {
+proc compute(ref arg) {
```

Comment:
Let's add something like `writeln("working with ", arg.type:string)` here so we can understand the test output better.

Reply:
OK, done.